src/javascript/app/pages/user/account/settings/account_closure.js

```diff
@@ -0,0 +1,93 @@
+const BinarySocket         = require('../../../../base/socket');
+const localize             = require('../../../../../_common/localize').localize;
+const Url                  = require('../../../../../_common/url');
+
+const AccountClosure = (() => {
+    const form_selector = '#form_closure';
+    let $txt_other_reason,
+        $closure_loading,
+        $closure_container,
+        $success_msg,
+        $error_msg;
+
+    const onLoad = () => {
+        $txt_other_reason  = $('#other_reason');
+        $closure_loading   = $('#closure_loading');
+        $closure_container = $('#closure_container');
+        $success_msg       = $('#msg_main');
+        $error_msg         = $('#msg_form');
+
+        $(form_selector).on('submit', (event) => {
```

Comment:
why not also store `$form` since it's also used on line 29? then line 29 can change to `$form.find('#btn_submit')`